javascript/src/URDFClasses.js

```diff
@@ -217,7 +217,8 @@ class URDFJoint extends URDFBase {
                 }
 
                 this.position.copy(this.origPosition);
-                this.position.addScaledVector(this.axis, pos);
+                const tempAxis = this.axis.clone().applyEuler(this.rotation);
+                this.position.addScaledVector(tempAxis, pos);
```

Comment:
This looks right - but can we add a global `Vector3` instance for this so we're not creating a new vector for the GC every time we modify the joint? This would just mean creating a global "_tempAxis" at the top of the file to copy the axis into before multiplying instead of cloning the axis field.

Reply:
Yes, just pushed that up! Thanks